src/goto-programs/restrict_function_pointers.cpp

```diff
@@ -291,6 +291,62 @@ function_pointer_restrictionst::parse_function_pointer_restrictions_from_file(
   return merged_restrictions;
 }
 
+static std::string resolve_pointer_name(
```

Comment:
A comment on the intended purpose/use-case for this new function might have been nice, but I also appreciate all you've done here is lift existing code out into a function, so I'm not going to hold it against you if you don't want/have time to add commentary. :-)

Reply:
Fair point, I've now added such a comment! Thank you for holding me to that bar.